### DISCUSSION TITLE
如何升级JumpServer版本从3.10.12到4.2

### DISCUSSION THREAD

**Discussion by DeepFal** [1Panel-dev/1Panel#6867] [Q&A]
如图在另一台机器上安装了1panel后看到jumpserver可安装的最新版本为4.2
![image](https://github.com/user-attachments/assets/facda106-4cd7-4ef5-a298-5172cc43f70e)
但是在已经安装JumpServer3.10.12的服务器上的可升级列表中没看到可以升级
![image](https://github.com/user-attachments/assets/944cd18a-6e71-40ea-af39-c553b6a90b67)
![image](https://github.com/user-attachments/assets/f2f648b1-f2ea-41b3-b4f9-19d295597ef4)

因为旧版的JumpServer服务器因oss地址失效导致无法正常初始化应用发布机导致被迫升级版本
求解如何从3.10.12升级到最新版本，谢谢

**Comment by wanghe-fit2cloud** [comment]
JumpServer 因为是跨大版本升级，在本地测试时容易出现升级失败的情况，所以在 1Panel 应用商店中同时提供了 v3 和 v4 两个版本，但不支持跨版本升级，仅支持全新安装。

**Comment by wanghe-fit2cloud** [comment]
具体问题，还可以咨询 JumpServer 项目组。
https://github.com/jumpserver/jumpserver
